src/main/resources/assets/computercraft/lua/rom/motd.txt

```diff
@@ -13,3 +13,22 @@ Use the "edit" program to create and edit your programs.
 You can copy files with "copy" or "cp".
 You can use "wget run <url>" to run a program from the internet.
 You can use "wget" to download a file from the internet.
+Use "fg" or "bg" to run multiple programs at the same time.
```

Comment:
Probably should mention that this requires mutishell which in turn needs and advanced computer

Reply:
Ithink that would be to long. If the User run fg or bg without multishell or advanced computer he get a error message that told him what he need to run.

Reply:
Perhaps something like this? `On an advanced computer you can use "fg" or "bg" to run multiple programs at the same time.`

Reply:
@lupus590 I have used your sentence now.